### PR TITLE
fix & docs (tutorials): fix aat header violation and content updates

### LINKED ISSUE #2028: audit-avt1-all-about-carbon-partners: h3 element id carbon-website
The `<h3>` element has the id "the-people" that is already in use 

Screen Shot 2020-12-03 at 10 48 35 AM

### PR DESCRIPTION
**Pull request by andreancardona**
Docs update & accessibility fix

You should see the following changes:

Angular Tutorial - Wrapping up: 
- Badge Information --> Badge process
- `IBM Digital badge`(underneath the badge image) to a lower case “b” for badge.
- Last ## Application form (title deleted)

React Tutorial - Wrapping up: 
- Badge Information --> Badge process
- `IBM Digital badge`(underneath the badge image) to a lower case “b” for badge.
- Last ## Application form (title deleted)

Vue Tutorial - Wrapping up: 
- Badge Information --> Badge process
- `IBM Digital badge`(underneath the badge image) to a lower case “b” for badge.
- Last ## Application form (title deleted)

## Testing:
- [ ] Run accessibility checker on the following pages and make sure the following error does not show up: 

- [Angular Tutorial - Wrapping up](https://carbondesignsystem-git-2028-aat-partners-h3-id.carbon-design-system.now.sh/developing/angular-tutorial/wrapping-up)
- [React Tutorial - Wrapping up](https://carbondesignsystem-git-2028-aat-partners-h3-id.carbon-design-system.now.sh/developing/react-tutorial/wrapping-up)
- [Vue Tutorial - Wrapping up](https://carbondesignsystem-git-2028-aat-partners-h3-id.carbon-design-system.now.sh/developing/vue-tutorial/wrapping-up)

Screen Shot 2020-12-04 at 8 57 19 AM

### CODE REVIEW

**Comment by janchild** [comment]
@mattrosno We needed to remove the duplication of headings on these pages. Could you please check and make sure you're okay with my suggestions. Thanks!

File: src/pages/developing/angular-tutorial/wrapping-up.mdx [hunk 1]
```diff
@@ -33,7 +33,7 @@ available to anybody who completes the tutorial, not just IBM employees.
 
 ![](/images/carbon-badge.png)
 
-<Caption>IBM Digital Badge</Caption>
+<Caption>IBM Digital badge</Caption>
```
Comment (mattrosno) [inline src/pages/developing/angular-tutorial/wrapping-up.mdx:36]: Why update casing here but not consistently across the entire page? There are other uses of "IBM Digital Badge".

File: src/pages/developing/react-tutorial/wrapping-up.mdx [hunk 2]
```diff
@@ -18,7 +18,7 @@ Badge.
 <AnchorLinks>
 
 <AnchorLink>Application form</AnchorLink>
-<AnchorLink>Badge information</AnchorLink>
+<AnchorLink>Badge process</AnchorLink>
```
Comment (mattrosno) [inline src/pages/developing/react-tutorial/wrapping-up.mdx:21]: This link is broken because the heading is still "Badge information" below?

**Review by mattrosno — COMMENTED** [review]
I see broken anchor links and I don't understand the selective case changes to "IBM Digital badge".

File: src/pages/developing/angular-tutorial/wrapping-up.mdx (hunk 1 above)
Comment (andreancardona) [reply src/pages/developing/angular-tutorial/wrapping-up.mdx:36]: @janchild Any thoughts here?

File: src/pages/developing/react-tutorial/wrapping-up.mdx (hunk 2 above)
Comment (andreancardona) [reply src/pages/developing/react-tutorial/wrapping-up.mdx]: resolved. :)

File: src/pages/developing/angular-tutorial/wrapping-up.mdx (hunk 1 above)
Comment (janchild) [reply src/pages/developing/angular-tutorial/wrapping-up.mdx:36]: @andreancardona @mattrosno I'll go through the file and make sure it's consistent.

Comment (janchild) [inline src/pages/developing/angular-tutorial/wrapping-up.mdx:36]:
```suggestion
<Caption>IBM Digital Badge</Caption>
```

File: src/pages/developing/react-tutorial/wrapping-up.mdx [hunk 3]
```diff
@@ -33,7 +33,7 @@ available to anybody who completes the tutorial, not just IBM employees.
 
 ![](/images/carbon-badge.png)
 
-<Caption>IBM Digital Badge</Caption>
+<Caption>IBM Digital badge</Caption>
```
Comment (janchild) [inline src/pages/developing/react-tutorial/wrapping-up.mdx:36]:
```suggestion
<Caption>IBM Digital Badge</Caption>
```

File: src/pages/developing/react-tutorial/wrapping-up.mdx [hunk 4]
```diff
@@ -53,7 +53,8 @@ badge:
    - Step 5. [Deploying to IBM Cloud](/developing/react-tutorial/step-5)
 2. Have approved pull requests for steps 1 through 5 in the
    [carbon-tutorial repository](https://github.com/carbon-design-system/carbon-tutorial)
-3. Complete the [badge application](https://badges.carbondesignsystem.com)
+3. Apply for the IBM Digital Badge by completing the
+   [form and survey](https://badges.carbondesignsystem.com/)
```
Comment (janchild) [inline src/pages/developing/react-tutorial/wrapping-up.mdx:57]:
```suggestion
   [form and survey](https://badges.carbondesignsystem.com/).
```

File: src/pages/developing/react-tutorial/wrapping-up.mdx [hunk 5]
```diff
@@ -53,7 +53,8 @@ badge:
    - Step 5. [Deploying to IBM Cloud](/developing/react-tutorial/step-5)
 2. Have approved pull requests for steps 1 through 5 in the
    [carbon-tutorial repository](https://github.com/carbon-design-system/carbon-tutorial)
```
Comment (janchild) [inline src/pages/developing/react-tutorial/wrapping-up.mdx:55]:
```suggestion
   [carbon-tutorial repository](https://github.com/carbon-design-system/carbon-tutorial).
```

File: src/pages/developing/vue-tutorial/wrapping-up.mdx [hunk 6]
```diff
@@ -33,7 +33,7 @@ available to anybody who completes the tutorial, not just IBM employees.
 
 ![](/images/carbon-badge.png)
 
-<Caption>IBM Digital Badge</Caption>
+<Caption>IBM Digital badge</Caption>
```
Comment (janchild) [inline src/pages/developing/vue-tutorial/wrapping-up.mdx:36]:
```suggestion
<Caption>IBM Digital Badge</Caption>
```

File: src/pages/developing/vue-tutorial/wrapping-up.mdx [hunk 7]
```diff
@@ -53,7 +53,8 @@ badge:
    - Step 5. [Deploying to IBM Cloud](/developing/vue-tutorial/step-5)
 2. Have approved pull requests for steps 1 through 5 in the
    [carbon-tutorial-vue repository](https://github.com/carbon-design-system/carbon-tutorial-vue)
-3. Complete the [badge application](https://badges.carbondesignsystem.com)
+3. Apply for the IBM Digital Badge by completing the
+   [form and survey](https://badges.carbondesignsystem.com/)
```
Comment (janchild) [inline src/pages/developing/vue-tutorial/wrapping-up.mdx:57]:
```suggestion
   [form and survey](https://badges.carbondesignsystem.com/).
```

File: src/pages/developing/vue-tutorial/wrapping-up.mdx [hunk 8]
```diff
@@ -53,7 +53,8 @@ badge:
    - Step 5. [Deploying to IBM Cloud](/developing/vue-tutorial/step-5)
 2. Have approved pull requests for steps 1 through 5 in the
    [carbon-tutorial-vue repository](https://github.com/carbon-design-system/carbon-tutorial-vue)
```
Comment (janchild) [inline src/pages/developing/vue-tutorial/wrapping-up.mdx:55]:
```suggestion
   [carbon-tutorial-vue repository](https://github.com/carbon-design-system/carbon-tutorial-vue).
```

**Review by janchild — COMMENTED** [review]
@andreancardona I have some tweaks here. I need to change a couple of other things. Will ping you in Slack.

**Review by dakahn — APPROVED** [review]

**Review by janchild — APPROVED** [review]
LGTM!

**Review by joshblack — APPROVED** [review]

**Pull request merged**